src/lib_c/aarch64-linux-gnu/c/sys/types.cr

```diff
@@ -7,7 +7,7 @@ lib LibC
   alias ClockT = Long
   alias ClockidT = Int
   alias DevT = ULong
-  alias GidT = UInt
+  alias GidT = UInt32
```

Comment:
These were automatically generated, they should be correct.

Reply:
They may be auto-generated but I don't think they are. I dug through the source/includes of OpenBSD 6.2 (current), FreeBSD 11 (Current), must (current), and Debian 9 (current) for the basis of my changes.

Reply:
There is no `LibC::UInt32` type. The correct C type is `LibC::UInt`.

Reply:
I made a typo, it should be `UInt32T`, which is now fixed. But, they are uint32_t not UInt. I also noticed that `stdint.cr` defines `alias UInt32T = UInt`. This isn't correct. The `int<width>_t` and `uint<width>_t` types were created for the explicit purpose of removing this type of ambiguity. 

FreeBSD 11.1 - /user/include/sys/types.h:123
```
typedef	__gid_t		gid_t;		/* group id */
```
FreeBSD 11.1 - /user/include/sys/_types.h:44
```
typedef	__uint32_t	__gid_t;
```
---
OpenBSD 6.2 - /user/include/sys/types.h:139
```
typedef	__gid_t		gid_t;		/* group id */
```
OpenBSD 6.2 - /user/include/sys/_types.h:48
```
typedef	__uint32_t	__gid_t;	/* group id */
```
---
Darwin - /user/include/sys/types.h:109
```
#include <sys/_types/_gid_t.h>
```
Darwin - /user/include/sys/_gid_t.h:31
```
typedef __darwin_gid_t	gid_t; 
```
Darwin - /user/include/sys/_types.h:44
```
typedef __uint32_t	__darwin_gid_t;		/* [???] process and group IDs */
```